addons/web/static/src/js/framework/misc.js

```diff
@@ -172,6 +172,12 @@ function ReloadContext (parent, action) {
 }
 core.action_registry.add("reload_context", ReloadContext);
 
+function SuccessNotify (parent, action) {
```

Comment:
looking good, but you should add some docstring.  Here is a suggestion:

/**
 * This client action simply displays a notification. It is useful when we
 * do an action (such as a server action), that does not need to do anything,
 * except give some feedback to the user.
 *
 * @param {ActionManager} parent
 * @param {Object} action
 */